scripts/apps/vocabularies/views/vocabulary-config-modal.html

```diff
@@ -288,4 +288,72 @@ <h3 class="modal__heading" ng-show="!vocabulary._id"><span translate>Create</spa
             </button>
         </div>
     </div>
+
+    <div sd-modal data-model="vocabulary" class="modal--large" ng-if="vocabulary.field_type === 'upload' ">
```

Comment:
would be good to have this part in a standalone component so we can reuse it in other places later (eg for content types)

Reply:
I agree. And let's write new code in react instead of angular